### PR TITLE
Model

### PR DESCRIPTION
**Pull request by 11TK11**
class with path method of search

### CODE REVIEW

File: src/main/com/fundations/convertor/model/Search.java [hunk 1]
```diff
@@ -10,18 +10,29 @@
...
  */
 
-package main.com.fundations.convertor;
+package main.com.fundations.convertor.model;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
 /**
```
Comment (anghelitalo) [inline src/main/com/fundations/convertor/model/Search.java]: Maybe the head comment should go to the first lines and before the imports.

File: src/main/com/fundations/convertor/model/Search.java [hunk 2]
```diff
@@ -10,18 +10,29 @@
...
  *  Search class for java applications, class which search videos
  *
  * @author Kevin Sanchez - AWT-[01].
  * @version 0.1
  */
 public class Search {
-    public Search() {
+    private List<String> lFicheros=new ArrayList<String>();
+    private String ruta;
```
Comment (anghelitalo) [inline src/main/com/fundations/convertor/model/Search.java]: The names of variables, are in spanish. Maybe the notation should be use all in english.

File: src/main/com/fundations/convertor/model/Search.java [hunk 3]
```diff
@@ -10,18 +10,29 @@
...
+    public Search(String ruta) {
+        this.ruta = ruta;
     }
-    public int Found(){
-        
-        return 0;
+    public List<String> Found(){
+        File dir= new File(ruta);
+        String[] ficheros=dir.list();
+            for(int x=0;x<ficheros.length;x++)
```
Comment (anghelitalo) [inline src/main/com/fundations/convertor/model/Search.java]: In my opinion the variable X is like missing the meaning of the function, rename this X for other name.

File: src/main/com/fundations/convertor/model/Search.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+    public Search(String path) {
+        this.path = path;
+    }
+
+    /**
+     * this method allows to create a list of String with files
+     * from the given path, that list is returned to be used
+     */
+    public List<String> findPath(){
```
Comment (AaronClaros) [inline src/main/com/fundations/convertor/model/Search.java]: i suggest change the name of method findPath() to getAllFilesOnPath(), this in order to make more easy to understand what do this method

Comment (11TK11) [reply src/main/com/fundations/convertor/model/Search.java]: ok

**Pull request merged**